frontend/components/home/Faq.tsx

```
@@ -0,0 +1,631 @@
+                            </div>
+                            <div className='faq-img'>
+                                <Image src={faqImg} alt='faq' />
+                                <div className='shape-one'>
+                                    <svg
+                                        width='243'
+                                        height='201'
+                                        viewBox='0 0 243 201'
+                                        fill='none'
+                                        xmlns='http://www.w3.org/2000/svg'
+                                        className='injectable injectable tg-motion-effects4'
+                                    >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)